docs/quick-start/backend-service-tutorial.txt

```diff
@@ -0,0 +1,482 @@
+   :description: Learn how to set up a back end and perform CRUD operations by using Laravel MongoDB.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this tutorial, you create a simple REST back end for a front-end app
+by using {+odm-long+}. The tutorial uses Laravel's built-in API routing
+features.
+
+Prerequisites
+-------------
+
+Before you can start this tutorial, you need the following software
+installed in your development environment:
+
+- MongoDB Atlas cluster with sample data loaded. To learn how to create
+  a cluster, see the :ref:`laravel-quick-start-create-deployment` step
+  of the Quick Start guide.
+- `PHP <https://www.php.net/downloads>`__.
+- `Composer <https://getcomposer.org/doc/00-intro.md>`__.
+- `{+php-extension+} <https://www.mongodb.com/docs/php-library/current/get-started/#download-and-install>`__.
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a Laravel project.
+
+      First, create a Laravel project directory. Then, run the following
+      command to create a new Laravel project called ``laraproject``:
+      
+      .. code-block:: bash
+
+         composer create-project laravel/laravel laraproject
+
+   .. step:: Install {+odm-long+}.
+
+      To check that {+odm-long+} is running in the web server,
+      add a webpage to your Laravel website. In your project,
+      navigate to ``/routes/web.php`` and add the following route:
+
+      .. code-block:: php
+
+         Route::get('/info', function () {
+             phpinfo();
+         });
+
+      Then, run the following command in your shell to start
+      your application:
+
+      .. code-block:: bash
+
+         php artisan serve
+
+      After the application begins running, navigate to
+      http://127.0.0.1:8000/info to view the PHPinfo page. Scroll down
+      to or search for the **mongodb** entry to verify that
+      the {+php-extension+} is installed.
+
+      Run the following command in your shell to install {+odm-long+}:
+
+      .. code-block:: bash
+
+         composer require mongodb/laravel-mongodb:^{+package-version+}
+
+   .. step:: Configure your MongoDB connection.
+
+      Open your project's ``config/database.php`` file and update the
+      ``connections`` array as shown in the following code:
+
+      .. code-block:: php
+
+         'connections' => [
+                 'mongodb' => [
+                     'driver' => 'mongodb',
+                     'dsn' => env('MONGODB_URI', '<connection string>'),
+                     'database' => 'db',
+                 ],
+      
+      Ensure that you either replace the connection string placeholder
+      in the preceding code or set the ``MONGODB_URI`` environment variable to
+      your connection string before you run your application. To learn
```

Comment:
```suggestion
      in the preceding code or set the ``MONGODB_URI`` environment variable to
      your connection string and remove the placeholder before you run your application. To learn
```
S: Just because the code won't work as intended if you leave the placeholder if I'm understanding correctly.

Reply:
resolving as I used only the placeholder